src/Sentry.Log4Net/Sentry.Log4Net.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>$(SentryMainTargetFrameworks)</TargetFrameworks>
```

Comment:
This is bad because now u're adding a bunch of new TFMs on a "lagacy" package

Reply:
```suggestion
    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
```